### PR TITLE
feat: how to get new API tokens

### PR DESCRIPTION
**Pull request by IC-1101asterisk**

### CODE REVIEW

File: docs/source/documentation/client_configuration.rst [hunk 1]
```diff
@@ -115,3 +115,12 @@ such as passwords or tokens in environment variables.
     )
 
 
+Generating new API tokens
+-----------------------------------------
+To do so you need to go to the API tokens management page on the frontend. Then you will see a list of your current token as well
```
Comment (RomainGoussault) [inline docs/source/documentation/client_configuration.rst:120]:
```suggestion
To do so you need to go to the API tokens management page on the web application. Then you will see a list of your current token as well
```

File: docs/source/documentation/client_configuration.rst [hunk 2]
```diff
@@ -115,3 +115,12 @@ such as passwords or tokens in environment variables.
...
+as an option to generate a new one. 
+
+Clicking on this button will open a menu allowing you to pick a name and an expiration date for
+your new token. 
+
+Afterward you will be able to see your token once, do copy it somewhere safe before proceeding with your work.
```
Comment (RomainGoussault) [inline docs/source/documentation/client_configuration.rst:126]:
```suggestion
Afterward you will be able to see your token only once. Do copy it somewhere safe before proceeding with your work. 
```

File: docs/source/documentation/client_configuration.rst [hunk 3]
```diff
@@ -115,3 +115,12 @@ such as passwords or tokens in environment variables.
     )
 
 
+Generating new API tokens
```
Comment (RomainGoussault) [inline docs/source/documentation/client_configuration.rst:118]: I am not sure about putting this piece of information on this page (and then renaming something more general "How to login"). Or using a second page "How to deal with API token"

Comment (RomainGoussault) [reply docs/source/documentation/client_configuration.rst:118]: I slightly favor the second option (new page) as it make each page clearer.
What do you think @IC-1101asterisk @SdgJlbl?

**Review by RomainGoussault — COMMENTED** [review]
Thanks for the PR!

**Comment by IC-1101asterisk** [comment]
@RomainGoussault if you had any comments on this

File: docs/source/documentation/client_configuration.rst (hunk 3 above)
Comment (IC-1101asterisk) [reply docs/source/documentation/client_configuration.rst:118]: I agree, I don't think this has anything to do with the rest of the page, it should be put somewhere else
Just waiting for @SdgJlbl to voice her opinion on this

Comment (SdgJlbl) [reply docs/source/documentation/client_configuration.rst:118]: Yes, I think it's better to keep how-to guides short and focused on only one thing at a time :) 
You can create a new page "How-to generate new API tokens for login"

File: docs/source/documentation/api_tokens_generation.rst [hunk 4]
```diff
@@ -0,0 +1,20 @@
+How-to generate new API tokens for login
+================================
+
+This short guide explains how to use the API tokens management page.
+
+Generating new API tokens
+-----------------------------------------
+To do so you need to go to the API tokens management page on the web application. Then you will see a list of your current token as well
```
Comment (SdgJlbl) [inline docs/source/documentation/api_tokens_generation.rst:8]:
```suggestion
To do so you need to go to the API tokens management page on the web application. You will see a list of your current token as well
```

File: docs/source/documentation/api_tokens_generation.rst [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+as an option to generate a new one. 
+
+Clicking on this button will open a menu allowing you to pick a name and an expiration date for
```
Comment (SdgJlbl) [inline docs/source/documentation/api_tokens_generation.rst:11]:
```suggestion
Clicking on this button opens a menu allowing you to pick a name and an expiration date for
```

File: docs/source/documentation/api_tokens_generation.rst [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+your new token. 
+
+Afterward you will be able to see your token only once. Do copy it somewhere safe before proceeding with your work.
```
Comment (SdgJlbl) [inline docs/source/documentation/api_tokens_generation.rst:14]:
```suggestion
Afterward your token will be shown only once. Do copy it somewhere safe before proceeding with your work. 
```

**Review by SdgJlbl — COMMENTED** [review]
Thanks for the PR 👍 
Don't forget to add a line in the changelog :)

**Review by RomainGoussault — COMMENTED** [review]
Thanks for the PR!

**Review by RomainGoussault — APPROVED** [review]

**Pull request merged**